uberduck_ml_dev/models/tacotron2.py

```diff
@@ -405,9 +405,9 @@ def inference_noattention(self, memory, attention_map):
         for i in range(len(attention_map)):
 
             attention = attention_map[i]
-            decoder_input = torch.cat((self.prenet(decoder_input)), dim=1)
+            decoder_input = torch.cat((self.prenet(decoder_input),), dim=1)
```

Comment:
Does this do anything?
![image](https://user-images.githubusercontent.com/85975676/148292242-da4e7ec6-f5b4-47f8-ba17-f43de471524e.png)